Description:Include <locale.h> in guvcview.c to avoild 
"undeclared LC_ALL" error when compiling.
Author: Xiangwei Gui <yuemeng.gui@gail.com>
--- a/guvcview/guvcview.c
+++ b/guvcview/guvcview.c
@@ -28,6 +28,7 @@
 #include <sys/stat.h>
 #include <sys/resource.h>
 #include <errno.h>
+#include <locale.h>
 
 #include "gview.h"
 #include "gviewv4l2core.h"
